### PR TITLE
fix: now the sidebar uses dvh for dynamic resizing of the browser

### PR DESCRIPTION
**Pull request by nickytonline**
## Description

Now the sidebar uses `dvh` units to adjust it's size for parts of the browser viewport that resize dynamically, e.g. address bar disappearing/appearing, etc.

I've tested this on an iPhone, but not Android. @adiati98, or someone with an Android phone, let me know if it resolves the issue. Thanks in advance!

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

Fixes the height issue in #2834.

## Mobile & Desktop Screenshots/Recordings

https://github.com/open-sauced/app/assets/833231/2fb8be88-4a94-4442-ace4-e7d88cd3bf3d

## Steps to QA

1. You need to test the deploy preview on an actual phone. Use the QR Code for the deploy preview.
2. Log in.
3. Click on the Workspace link up top.
4. The sidebar should adjust as you scroll and it should also take 100% height.

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

**Comment by adiati98** [comment]
@nickytonline it works! Thank you! 🎉

https://github.com/open-sauced/app/assets/45172775/bb8aa684-313f-4726-9e56-3e8f8260f7f1

File: components/shared/AppSidebar/AppSidebar.tsx [hunk 1]
```diff
@@ -52,12 +52,11 @@ interface AppSideBarProps {
 }
 
 export const AppSideBar = ({ workspaceId, hideSidebar, sidebarCollapsed }: AppSideBarProps) => {
-  const { user } = useSupabaseAuth();
   const { data: rawRepoInsights, isLoading: repoInsightsLoading } = useWorkspacesRepositoryInsights({ workspaceId });
   const { data: rawContributorInsights, isLoading: contributorInsightsLoading } = useWorkspacesContributorInsights({
     workspaceId,
   });
-  const { data: workspaces, isLoading: workspacesLoading, mutate } = useWorkspaces({ load: !!user, limit: 100 });
+  const { data: workspaces, isLoading: workspacesLoading, mutate } = useWorkspaces({ limit: 100 });
```
Comment (brandonroberts) [inline components/shared/AppSidebar/AppSidebar.tsx:59]: Why are these changes being reverted?

Comment (nickytonline) [reply components/shared/AppSidebar/AppSidebar.tsx:59]: Oh weird. I think that creeped in, when I was testing your PR. 😅 I'll revert that.

**Review by brandonroberts — APPROVED** [review]

**Review by jpmcb — APPROVED** [review]

**Review by zeucapua — APPROVED** [review]
the `v` added:
its-just-a-tiny-change-shea-whitney-gif

**Comment by isabensusan** [comment]
aah -- I was literally doing my review, it's a tiny bit buggy on Android, the sidebar doesn't go all the way down, but it's nothing we can't live without. This PR is correctly merged, we can continue to work on details

https://github.com/open-sauced/app/assets/27322879/2b756c41-1ca1-443a-b65a-e1262f6e3a4b

**Pull request merged**